userguide/advanceduse/dav.rst

```diff
@@ -14,22 +14,48 @@ These instructions work for CalDAV as well.
 1) Follow this `guide <https://docs.ubports.com/en/latest/userguide/advanceduse/adb.html>`_ to activate Developer Mode and a connection either through ADB or SSH.
 
 
-2) Download this `script <https://gist.github.com/bastos77/0c47a94dd0bf3e394f879c0ff42b7839>`_ (let's call it dav.sh) and edit the following variables: 
+2) OPTION A: 
 
-* Server side: CAL_URL, **CONTACTS_URL**, USERNAME, PASSWORD (of your ownCloud/nextCloud/baikal/SOGO/… server)
-* CONTACT and CALENDAR _ NAME / VISUAL_NAME / CONFIG_NAME (it's more cosmetic)
-* CRON_FREQUENCY (for the frequency of synchronization)
-* Line 61: write ``sudo sh -c "echo '$COMMAND_LINE' > /sbin/sogosync"`` , instead of ``sudo echo "$COMMAND_LINE" > /sbin/sogosync``, to avoid a "Permission denied" error.
+    2.1    Download this `script <https://gist.github.com/bastos77/0c47a94dd0bf3e394f879c0ff42b7839>`_ (let's call it dav.sh) and edit the following variables: 
 
-3) Move the file to your Ubuntu Touch device, either by way of a file manager or through the use of ADB:: 
+    * Server side: CAL_URL, **CONTACTS_URL**, USERNAME, PASSWORD (of your ownCloud/nextCloud/baikal/SOGO/… server)
```

Comment:
you mention "Server side", why not also "Phone". or drop it. It is confusing to have this *one* item only

why is CONTACTS_URL in bold? 

pls format all code like "CONTACTS_URL", "USERNAME", "dav.sh", etc as code with double backquotes ``like this`` 

depending what you do wrt to the list overall (remove top level?) it might be easier to read if you break out the items (CAL_URL, etc) into a sublist